libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -219,7 +214,7 @@ const AP_Param::GroupInfo NavEKF3::var_info[] = {
 
     // @Param: HGT_DELAY
     // @DisplayName: Height measurement delay (msec)
-    // @Description: This is the number of msec that the Height measurements lag behind the inertial measurements.
+    // @Description: This is the number of msec that the Height measurements lag behind the inertial measurements. The autpilot should be restarted after adjusting this parameter.
```

Comment:
Instead of all these comments in the description it should instead be added `@RebootRequired: True`